cmd/ak/common/flags.go

```diff
@@ -45,7 +45,10 @@ func ToExitCodeError(err error, what string) error {
 	case errors.Is(err, sdkerrors.ErrFailedPrecondition):
 		msg = fmt.Sprintf("on %s", what)
 		code = FailedPrecondition
-	}
+	case errors.As(err, resolver.NotFoundErrorType) {
```

Comment:
why errors.As instead of Is?

Reply:
I moved the code without changing the logic. So it was `errors.As` and since there is no tests for this I prefer to leave it as is